projects-yaml-reference_5b6813d20428631d7a89afa9.md

```diff
@@ -109,7 +111,56 @@ $ sem create -f goDemo.yaml
 error: http status 422 with message "{"message":"repository url must be an SSH url"}" received from upstream
 ```
 
-## Example
+#### schedulers
+
+The schedulers property can contain a list of schedulers defined on the
+project.
+
+A scheduler in Semaphore 2.0 is a way to run a pre-defined pipeline on a project
+at the pre-defined time.
+
+A scheduler has a few properties: `name`, `branch`, `at` and
+`pipeline_file`.
+
+##### name
+
+The `name` property uniquely identifies the scheduler for a project.
+
+##### branch
+
+The `branch` property specifies which currently existing branch will be used in
+running the pipeline.
```

Comment:
One limitation here is that there has to be at least one workflow initiated by push on that branch.
Maybe that should be added to docs also.